deepchem/models/torch_models/layers.py

```diff
@@ -7302,3 +7302,276 @@ def cosine_dist(x: torch.Tensor, y: torch.Tensor) -> torch.Tensor:
 
     cosine_similarity = torch.matmul(x_norm, y_norm.transpose(-1, -2))
     return cosine_similarity
+
+
+class BN(nn.Module):
```

Comment:
SE3EquivariantLayerNormalization? BN is a bit too short. Feel free to come up with a better name too